gradle.properties

```diff
@@ -14,4 +14,9 @@ org.gradle.jvmargs=-Xmx1536m
 # When configured, Gradle will run in incubating parallel mode.
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
-# org.gradle.parallel=true
+org.gradle.parallel=true
+org.gradle.caching=true
+
+VERSION_NAME=0.1.0
+VERSION_CODE=1
+GROUP=com.pusher
```

Comment:
Based on the README changes, does that mean that I could now just add a line like this: `pusher_platform_local=../pusher-platform-android` below this line, and it would all just work, without any extra config changes?

Reply:
Yeah, but this file is checked in so it's better to have it in `~/.gradle/gradle.properties` as it's a local one